TheWarWithin/MageArcane.lua

```diff
@@ -1497,9 +1541,12 @@ spec:RegisterAbilities( {
             applyBuff( "arcane_surge" )
             addStack( "clearcasting" )
             mana.regen = mana.regen * 5.25
+            -- trick addon into thinking you have enough mana to cast arcane blast right after, because in reality you do
+            gain ( (mana.modmax*0.25), "mana" )
```

Comment:
I think this `gain` will automatically trigger `forecastResources` so the next line may be removable (forecastResources is an expensive function).